vignettes/getting-started.Rmd

```diff
@@ -74,6 +92,18 @@ You'll notice that this set up gives us a list of formatted data. This contains
 str(gimap_dataset)
 ```
 
+```{r}
+gimap_dataset <- run_qc(gimap_dataset, params = list(cell_line = "pgPEN_HeLa"), overwrite = TRUE)
```

Comment:
Is the a reason that params needs to be a list if there's only one item in the list? This may make it confusing for users more so then if they can just provide a `plot_name = "pgPEN_HeLa"`. But you may have some bigger idea for this params list so trying to understand your thoughts here. 

I wouldn't encourage them to put overwrite = TRUE in the vignette. Start out with overwrite = FALSE as the default but don't encourage them override that default. 
```suggestion
gimap_dataset <- run_qc(gimap_dataset, params = list(cell_line = "pgPEN_HeLa"))
```

Reply:
I was just trying to make the code work/run without error. I flagged this in my notes as something that could be simplified later